src/lib.rs

```diff
@@ -207,6 +209,7 @@ const WALLET_KEYS_SEED_LEN: usize = 64;
 /// | `network`                   | Network::Bitcoin |
 /// | `listening_address`         | 0.0.0.0:9735     |
 /// | `default_cltv_expiry_delta` | 144              |
+/// | `log_level`                 | `Debug`          |
```

Comment:
nit: inconsistency with `Network::Bitcoin` and here. The former doesn't use ticks and the latter doesn't use the enum name prefix.